src/webgl/loading.js

```diff
@@ -331,7 +455,7 @@ function parseObj(model, lines) {
   if (model.vertexNormals.length === 0) {
     model.computeNormals();
   }
-
+  console.log(model.vertexColors);
```

Comment:
I assume this is for debugging, although we should probably take this out before merging.